### PR TITLE
Woman in Blue pack!

### PR DESCRIPTION
**Pull request by mikemayhemdev**
Much less complex than the Claw pack.

### CODE REVIEW

File: src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  },
+  "${ModID}:ToxicVial": {
+    "NAME": "Toxic Vial",
+    "DESCRIPTION": "Choose: Apply 3 Weak, or Apply 8 Poison. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Choose: Apply 3 Vulnerable, Apply 3 Weak, or Apply 8 Poison.",
+    "EXTENDED_DESCRIPTION": ["Apply 3 Vulnerable.", "Apply 3 Weak.","Apply 8 Poison."]
+  },
+  "${ModID}:AttackVial": {
+    "NAME": "Attack Vial",
+    "DESCRIPTION": "Choose 1 of 3 Attacks to gain. It costs 0 until played. NL Exhaust.",
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json:43]: To add into your hand?

File: src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    "EXTENDED_DESCRIPTION": ["Choose 1 of 3 Attacks to gain. It costs 0 until played."]
+  },
+  "${ModID}:ChaosVial": {
+    "NAME": "Chaos Vial",
+    "DESCRIPTION": "Play the top 3 cards of your draw pile. NL Exhaust."
+  },
+  "${ModID}:VialOfMemory": {
+    "NAME": "Vial of Memory",
+    "DESCRIPTION": "Choose a card in your discard pile and return it to your hand. It costs 0 until played. NL Exhaust.",
+    "UPGRADE_DESCRIPTION": "Choose a card in your discard pile and return it to your hand. It costs 0 until played. NL stslib:Exhaustive 2."
```
Comment (erasels) [inline src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json:54]: You seem to have defaulted to exhaustive for basically every upgrade. It's fine but maybe a little boring

File: src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json (hunk 1 above)
Comment (mikemayhemdev) [reply src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json:43]: I use Gain elsewhere (ghost claws in clawpack) to condense a full line's worth of characters down to four for better line spacing.  If needed we can change it, but I think 'gain' is a sensible shortening.  Obtain is the word the game prefers for permanent effects.

File: src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json (hunk 2 above)
Comment (mikemayhemdev) [reply src/main/resources/anniv5Resources/localization/eng/womaninbluepack/Cardstrings.json:53]: Fair, I overused it.  Changed four of the upgrades to not be exhaustive 2.

**Pull request merged**